api/src/main/java/uk/gov/dhsc/htbhf/claimant/converter/NewClaimDTOToClaimRequestConverter.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.dhsc.htbhf.claimant.converter;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+import uk.gov.dhsc.htbhf.claimant.entity.EligibilityOverride;
+import uk.gov.dhsc.htbhf.claimant.model.EligibilityOverrideDTO;
+import uk.gov.dhsc.htbhf.claimant.model.NewClaimDTO;
+import uk.gov.dhsc.htbhf.claimant.service.ClaimRequest;
+
+@Component
+@RequiredArgsConstructor
+public class NewClaimDTOToClaimRequestConverter {
+
+    private final ClaimantDTOToClaimantConverter claimantConverter;
+
+    public ClaimRequest convert(NewClaimDTO dto) {
+        Claimant claimant = claimantConverter.convert(dto.getClaimant());
+        EligibilityOverride eligibilityOverride = convertToEligibilityOverride(dto.getEligibilityOverride());
+
+        return ClaimRequest.builder()
+                .claimant(claimant)
+                .deviceFingerprint(dto.getDeviceFingerprint())
+                .webUIVersion(dto.getWebUIVersion())
+                .eligibilityOverride(eligibilityOverride)
+                .build();
+    }
+
+    private EligibilityOverride convertToEligibilityOverride(EligibilityOverrideDTO dto) {
+        return dto == null ? null : EligibilityOverride.builder()
```

Comment:
Can we update the formatting here please, the builder should really be on it's own line.

Reply:
Yes, I agree. As an alternative, I'd prefer it to look something like this:

```
    private EligibilityOverride convertToEligibilityOverride(EligibilityOverrideDTO dto) {
        if(dto == null) {
            return null;
        }
        return EligibilityOverride.builder()
                .eligibilityOutcome(dto.getEligibilityOutcome())
                .build();
    }
```
This then very clear what is going on and the alternate flow returns quickly.

Reply:
yes, agreed